visualization/dashboard/src/Localization/en.json

```diff
@@ -538,5 +540,17 @@
     },
     "GlobalOnlyChart": {
         "helperText": "Explore the top k important features that impact your overall model predictions. Use the slider to show descending feature importances."
+    },
+    "ExplanationSummary": {
+        "whatDoExplanationsMean": "What do these explanations mean?",
+        "clickHere": "Learn more",
+        "shapTitle": "Shapley values",
+        "shapDescription": "This explainer uses SHAP, which is a game-theory approach to explaining models, where the importance of features sets is measured by \"hiding\" those features from the model through marginalization. Click the link below to learn more.",
+        "limeTitle": "LIME (Local Interpretable Model-Agnostic Explanations)",
+        "limeDescription": "This explainer uses LIME, which provides a linear approximation of the model. To get an explanation, we do the following: perturb the instance, get model predictions, and use these predictions as labels to learn a sparse linear model that is locally faithful. The weights of this linear model are used as 'feature importances'. Click the link below to learn more.",
```

Comment:
"to learn a sparse linear model" it's not always sparse, right?  Or am I misreading something?

Reply:
that is the text provided by Marco, I just pasted it :) Happy to change it as needed.

Reply:
ok I guess we can leave it then